### PR TITLE
[Darc-Update] Update dependencies from build 20181213.4 of https://github.com/dotnet/arcade

### PR DESCRIPTION
**Pull request by ViktorHofer**

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>52fab5d72c6271f56121695fe780f6893d3ed7bd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.CoreFxTesting" Version="1.0.0-beta.18612.9">
+    <Dependency Name="Microsoft.DotNet.CoreFxTesting" Version="1.0.0-beta.18613.3">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>52fab5d72c6271f56121695fe780f6893d3ed7bd</Sha>
```
Comment (safern) [inline eng/Version.Details.xml]: You might want to update the Sha

**Comment by safern** [comment]
It would be great if we do a full DARC update as part of these. I don't think it is super hard. If this is urgent, no worries.

File: eng/Version.Details.xml (hunk 1 above)
Comment (ViktorHofer) [reply eng/Version.Details.xml]: I never update the Sha, DARC can do that later for me.

**Comment by ViktorHofer** [comment]
@dotnet-bot test UWP NETNative x86 Release Build

**Pull request merged**